modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/MultiGeometryTypeBinding.java

```diff
@@ -112,7 +112,7 @@ public Object parse(ElementInstance instance, Node node, Object value)
     
     @Override
     public Object getProperty(Object object, QName name) throws Exception {
-        if (GML.geometryMember.equals(name)) {
+        if ("geometryMember".equals(name.getLocalPart())) {
```

Comment:
Bare string "geometryMember"? How about `GML.geometryMember.getLocalPart().equals(name.getLocalPart())`? Is this change to make it work for both GML 3.1 and 3.2?